pkg/gui/containers_panel.go

```diff
@@ -90,6 +94,37 @@ func (gui *Gui) handleContainerSelect(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) renderContainerEnv(container *commands.Container) error {
+	mainView := gui.getMainView()
+	mainView.Autoscroll = false
+	mainView.Wrap = gui.Config.UserConfig.Gui.WrapMainPanel
+	outputList := []string{""}
+	envMapping := map[string]string{}
+	padding := 0
+	if len(container.Details.Config.Env) > 0 {
```

Comment:
We can use the `utils.RenderTable` function here to handle the padding. It takes a slice of string slices, so we can pass it e.g. `[['MY_ENV', 'MY_ENV_VALUE'], ['MY_OTHER_ENV', 'MY__OTHER_ENV_VALUE']]` and it'll handle the padding. I think you can colour the strings beforehand but if not you can use `utils.RenderList` which has some example uses lying around as well

Reply:
Thanks for your time
I have made changes to use `utils.RenderTable` function with coloring strings beforehand
I have also made changes to `utils.RenderTable` method to not to panic when an empty array is passed to it
Please review the changes